decidim-core/db/migrate/20170809084005_add_scope_enabled_to_participatory_processes.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddScopeEnabledToParticipatoryProcesses < ActiveRecord::Migration[5.1]
+  def change
+    add_column :decidim_participatory_processes, :scope_enabled, :boolean, null: false, default: true
```

Comment:
I'd `s/scope_enabled/scopes_enabled`, it's clearer to me since what you are enabling is the whole "scopes" feature 😕